mantidproject/mantid
# Muon model fit range updates on changing data

**Description of work.**
When the user changed the data to analyse in the model analysis tab, the fitting range would not update. This would lead to frustration as the fit may not be possible (the range and data do not overlap). 



**To test:**


Enable the model analysis tab and plot
Open Muon Analysis
Load a range of runs (e.g. HIFI 84447-9)
Go to fitting, tick simultaneous
Add a function (linear background)
Go to the sequential tab and fit to all
Go to the results tab
Select some temperature and field sample logs
Generate the results table
Go to the model tab
Check that the start and end x contain the current data (compare to plot)
Change the x axis to a different log/parameter
Check that the start and end x have updated to contain the new data


Fixes #32792. 



In the release notes


---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
